packages/shared/src/hooks/usePersonalizedDigest.ts

```diff
@@ -51,16 +55,21 @@ export const usePersonalizedDigest = (): UsePersonalizedDigest => {
   );
 
   const { mutateAsync: subscribePersonalizedDigest } = useMutation(
-    async () => {
+    async (params: { hour?: number; type?: UserPersonalizedDigestType }) => {
+      const { hour, type } = params || {
+        hour: 8,
+        type: UserPersonalizedDigestType.Digest,
+      };
       const result = await request<
         {
           subscribePersonalizedDigest: UserPersonalizedDigest;
         },
         Partial<UserPersonalizedDigestSubscribe>
       >(graphqlUrl, SUBSCRIBE_PERSONALIZED_DIGEST_MUTATION, {
         day: 3,
-        hour: 8,
+        hour,
         timezone: user?.timezone || undefined,
```

Comment:
Added for removal here https://github.com/dailydotdev/apps/pull/2914

Reply:
Removed here now as well